docs/src/main/asciidoc/continuous-testing.adoc

```diff
@@ -96,12 +96,13 @@ The following commands are available:
 [f] - Re-run failed tests
 [b] - Toggle 'broken only' mode, where only failing tests are run (disabled)
 [v] - Print failures from the last test run
-[o] - Toggle test output (disabled)
 [p] - Pause tests
+[o] - Toggle test output (disabled)
 [i] - Toggle instrumentation based reload (disabled)
 [l] - Toggle live reload (enabled)
-[s] - Force live reload scan
+[s] - Force restart with any changes
```

Comment:
I'm not sure the "with any changes" brings anything. "Force restart" seems clearer to me.